pkgs/os-specific/linux/hid-fanatecff/default.nix

```
@@ -0,0 +1,38 @@
+{ lib, stdenv, kernel, fetchFromGitHub, }:
+
+stdenv.mkDerivation rec {
+  pname = "hid-fanatecff";
+  version = "0.1.0";
+
+  src = fetchFromGitHub {
+    owner = "gotzl";
+    repo = "hid-fanatecff";
+    rev = "0.1";
```

```suggestion
    rev = "refs/tags/${lib.versions.majorMinor version}";
```

Maybe we can use `lib.versions.majorMinor` here to maintain the version number in one place only.
What do you think?